### DISCUSSION TITLE
Request A Small Change

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#287] [Ideas]
Hijacking my own old post.. Lol.

Would like to request an addition. Can you add a config option to limit values of temp/percentages to 99? The reason for this is mostly aesthetics. Due to how the text aligns the numbers.  Why I think this is ok. First 99% is really the same as 100% in the grand scheme of things when it pertains to temp/usage percent. As for temps, if you are running over 80 you're doing it wrong. LOL so capping that at 99 should also not be an issue.

Thanks.

**Comment by c0ldJS** [comment]
> "Can you add a config option to limit values of temp/percentages to 99?"

`MAX_VALUE` covers this for everything except `TEXT`. And text displaying numbers at/higher than 100 has value (monitoring RPM on fans, for example).

> "... 99% is really the same as 100% in the grand scheme of things..."

Except when it isn't. 1% remaining on 10TB of storage is ~100GB which, sure, it's probably time to clean things up but there's still ample room.

> "As for temps, if you are running over 80 you're doing it wrong. LOL"

I dunno man... throw a beefy transcode job at an AM5 that has moderate cooling and watch as the temperature crosses 90°C. That's still within operational expectations and thermal limiting hasn't kicked in yet.
